Packages/Sites/Neos.NeosIo/Resources/Private/TypoScript/NodeTypes/Reference.List.ts2

```diff
@@ -1,6 +1,6 @@
 prototype(Neos.NeosIo:Reference.List) < prototype(TYPO3.Neos:Content) {
 	references = TYPO3.TypoScript:Collection {
-		collection = ${Search.query(site).nodeType('Neos.NeosIo:Reference').sortDesc('datePublished').execute()}
+		collection = ${Search.query(site).nodeType('Neos.NeosIo:Reference.ShowCase').sortDesc('datePublished').limit(500).execute()}
```

Comment:
It is on purpose to select all references and not only show cases, to have a mixed listing with show cases and case studies later. But as we don't have dynamic case studies currently it doesn't matter for now.